libcontainer/container_linux.go

```diff
@@ -836,6 +836,80 @@ func (c *linuxContainer) handleCriuConfigurationFile(rpcOpts *criurpc.CriuOpts)
 	}
 }
 
+func (c *linuxContainer) handleCheckpointingExternalNamespaces(rpcOpts *criurpc.CriuOpts, minVersion int, t configs.NamespaceType) error {
+	var nsType string
+	if t == configs.NEWNET {
```

Comment:
to me, switch / case / case / default ... looks a tad better than if / else if / else if / else, but up to you.